### PR TITLE
Made python quickstart README.md match the .NET purchasing example

### PR DESCRIPTION
**Pull request by nyemade-uversky**
# Description

Changed item purchase to 10 cars instead of 11 for better parity between language sdks.

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [ ] You've updated the quickstart's README if necessary
* [ ] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

**Review by greenie-msft — APPROVED** [review]

**Review by greenie-msft — APPROVED** [review]
LGTM

File: workflows/python/sdk/README.md
Comment (paulyuk) [inline workflows/python/sdk/README.md:1]: Thanks for teeing this up @nyemade-uversky 

Just need to fix line 35 to match cars left in stock.  That's python validation test fails. 
Before you PR pls try a `make validate` in the same folder as the makefile to check if tests will pass, and you'll catch lots of little errors.

**Review by paulyuk — CHANGES_REQUESTED** [review]
please see comments

File: workflows/python/sdk/README.md
Comment (paulyuk) [reply workflows/python/sdk/README.md:1]: Also if easy could you please pull against the release-1.11 branch?

**Pull request merged**